File: SDKs/nbg.gr/v3.1.5/java/lib/src/main/java/openapisdk/models/shared/ProprietaryBankTransactionCodeStructure1.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * ProprietaryBankTransactionCodeStructure1
 * Set of elements to fully identify a proprietary bank transaction code.
**/
public class ProprietaryBankTransactionCodeStructure1 {
    @JsonProperty("Code")
    public String code;
    public ProprietaryBankTransactionCodeStructure1 withCode(String code) {
        this.code = code;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Issuer")
    public String issuer;
    public ProprietaryBankTransactionCodeStructure1 withIssuer(String issuer) {
        this.issuer = issuer;
        return this;
    }
}

File: SDKs/nbg.gr/v3.1.5/python/src/sdk/models/shared/proprietarybanktransactioncodestructure1.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class ProprietaryBankTransactionCodeStructure1:
    r"""ProprietaryBankTransactionCodeStructure1
    Set of elements to fully identify a proprietary bank transaction code.
    """
    
    code: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('Code') }})
    issuer: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Issuer') }})
    
